_scalingo_api/events.md

```diff
@@ -1211,11 +1211,19 @@ _When:_ Each time an SCM integration link is created to an app
 `type=link_scm`
 
 {:.table}
-| field           | type   | description                                 |
-| --------------- | ------ | ------------------------------------------- |
-| repo_name       | string | Name of the repository                      |
-| linker_username | string | Username of the linker                      |
-| source          | string | Source URL of the repository                |
+| field                        | type    | description                                                                                          |
+| ---------------------------- | ------- | ---------------------------------------------------------------------------------------------------- |
+| repo_name                    | string  | Name of the repository                                                                               |
+| linker_username              | string  | Username of the linker                                                                               |
+| source                       | string  | Source URL of the repository                                                                         |
+| branch                       | string  | Branch on which the repository is tied                                                               |
+| auto_deploy                  | boolean | Auto-deploy the app when the parametered branches is updated                                         |
```

Comment:
Since the link is made with one branch, it must be a singular: "Auto-deploy the app when the parametered branch is updated".

Reply:
Right as well, fixing it